ClinGen/clincoded
# #1917: Display terms for HPO IDs in Case/Seg tables

Issue: https://github.com/ClinGen/clincoded/issues/1917

**Steps to test:**

1. New Variant Curation, start an Interpretation
2. Navigate to Case/Segregation panel, save evaluation for any criteria, and click "Add Evidence" to bring up modal
3. Enter any PMID, preview and click "Next"
4. Enter a comma separated list of HPO IDs for the "**Phenotypic feature(s) associated with proband(s) (HPO) (PP4)**" input (i.e. HP:0010010, HP:0010704, HP:0030300), use Get Terms button
      * Form validation present for HPOs via "Get Terms" and "Submit", cannot fetch or save unless HPO ID format is correct and is a comma separated list.
      * Get Terms button fetches the terms for the respective IDs and saves as an object to be passed to Case/Segregation tables, terms are cleared using red "Clear Terms" button
      * Duplicates are omitted
5. Feel free to fill in another inputs, otherwise, click "Submit". Redirected to Case/Segregation panel, IDs with the respective terms are displayed in gray "Master Table" cell and in "Curated Evidence (Specificity of phenotype)" towards bottom of page.
6. Edit the Evidence in either table, and the terms should still be displayed to the right of input, "Get Terms" button still disabled.
